CONTRIBUTING.md

```diff
@@ -0,0 +1,235 @@
+## Contributing to `freshli`
```

Comment:
I like the idea of having a Contributing file for how to setup your Dev environment, building, branching, etc, and leaving the ReadMe for consumers of the NuGet package.

Reply:
Removed this section.